plugin/src/main/resources/META-INF/plugin.xml

```diff
@@ -185,7 +185,9 @@
                          displayName="Test maverick"
                          shortName="TestMaverickTestSmellPytestInspection"/>
     </extensions>
-
+    <extensions defaultExtensionNs="com.intellij">
+        <appStarter implementation="org.jetbrains.research.pynose.headless.HeadlessRunner"/>
```

Comment:
We don't have in `master`, do we? Why is it here O_o?

Reply:
we do have it in master :')
we only removed ignored, unknown and general fixture inspections

Reply:
oh wait no we don't, sorry